### ISSUE TITLE
Cilium 1.12.2 or 1.12.3 cilium-agent crashloopbackoff on some nodes. 

### ISSUE THREAD

**Issue by binhnguyenduc** [cilium/cilium#21829]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### What happened?

Some cilium-agent failed to init daemon on some nodes, but init on other nodes. The nodes are identical. 

I installed Kubernetes with Kubesphere (use kubeadm behind the scene). Have encountered the issue with physical nodes, as well as virtual machine (vSphere). In case of virtual machine, the nodes are cloned but some still fail to init. 

The sysdump is from a new cluster, and the node where I got the log from (prod-ceph-02) has been used in another cluster before and worked fine. 

### Cilium Version

cilium-cli: v0.12.4 compiled with go1.19.1 on linux/amd64
cilium image (default): v1.12.2
cilium image (stable): v1.12.3
cilium image (running): v1.12.3

### Kernel Version

Linux prod-ceph-02 5.4.17-2136.311.6.1.el7uek.x86_64 #2 SMP Thu Sep 22 19:51:14 PDT 2022 x86_64 x86_64 x86_64 GNU/Linux

### Kubernetes Version

Client Version: version.Info{Major:"1", Minor:"22", GitVersion:"v1.22.10", GitCommit:"eae22ba6238096f5dec1ceb62766e97783f0ba2f", GitTreeState:"clean", BuildDate:"2022-05-24T12:56:35Z", GoVersion:"go1.16.15", Compiler:"gc", Platform:"linux/amd64"}
Server Version: version.Info{Major:"1", Minor:"22", GitVersion:"v1.22.10", GitCommit:"eae22ba6238096f5dec1ceb62766e97783f0ba2f", GitTreeState:"clean", BuildDate:"2022-05-24T12:50:52Z", GoVersion:"go1.16.15", Compiler:"gc", Platform:"linux/amd64"}

### Sysdump

sysdump too large so I uploaded it here: https://we.tl/t-ICrfcKIlVM

### Relevant log output

```shell
level=info msg="Started gops server" address="127.0.0.1:9890" subsys=daemon
level=info msg="Memory available for map entries (0.003% of 269575847936B): 673939619B" subsys=config
level=info msg="option bpf-ct-global-tcp-max set by dynamic sizing to 2364700" subsys=config
level=info msg="option bpf-ct-global-any-max set by dynamic sizing to 1182350" subsys=config
level=info msg="option bpf-nat-global-max set by dynamic sizing to 2364700" subsys=config
level=info msg="option bpf-neigh-global-max set by dynamic sizing to 2364700" subsys=config
level=info msg="option bpf-sock-rev-map-max set by dynamic sizing to 1182350" subsys=config
level=info msg="  --agent-health-port='9879'" subsys=daemon
level=info msg="  --agent-labels=''" subsys=daemon
level=info msg="  --agent-not-ready-taint-key='node.cilium.io/agent-not-ready'" subsys=daemon
level=info msg="  --allocator-list-timeout='3m0s'" subsys=daemon
level=info msg="  --allow-icmp-frag-needed='true'" subsys=daemon
level=info msg="  --allow-localhost='auto'" subsys=daemon
level=info msg="  --annotate-k8s-node='false'" subsys=daemon
level=info msg="  --api-rate-limit=''" subsys=daemon
level=info msg="  --arping-refresh-period='30s'" subsys=daemon
level=info msg="  --auto-create-cilium-node-resource='true'" subsys=daemon
level=info msg="  --auto-direct-node-routes='false'" subsys=daemon
level=info msg="  --bgp-announce-lb-ip='false'" subsys=daemon
level=info msg="  --bgp-announce-pod-cidr='false'" subsys=daemon
level=info msg="  --bgp-config-path='/var/lib/cilium/bgp/config.yaml'" subsys=daemon
level=info msg="  --bpf-ct-global-any-max='262144'" subsys=daemon
level=info msg="  --bpf-ct-global-tcp-max='524288'" subsys=daemon
level=info msg="  --bpf-ct-timeout-regular-any='1m0s'" subsys=daemon
level=info msg="  --bpf-ct-timeout-regular-tcp='6h0m0s'" subsys=daemon
level=info msg="  --bpf-ct-timeout-regular-tcp-fin='10s'" subsys=daemon
level=info msg="  --bpf-ct-timeout-regular-tcp-syn='1m0s'" subsys=daemon
level=info msg="  --bpf-ct-timeout-service-any='1m0s'" subsys=daemon
level=info msg="  --bpf-ct-timeout-service-tcp='6h0m0s'" subsys=daemon
level=info msg="  --bpf-ct-timeout-service-tcp-grace='1m0s'" subsys=daemon
level=info msg="  --bpf-filter-priority='1'" subsys=daemon
level=info msg="  --bpf-fragments-map-max='8192'" subsys=daemon
level=info msg="  --bpf-lb-acceleration='disabled'" subsys=daemon
level=info msg="  --bpf-lb-affinity-map-max='0'" subsys=daemon
level=info msg="  --bpf-lb-algorithm='random'" subsys=daemon
level=info msg="  --bpf-lb-dev-ip-addr-inherit=''" subsys=daemon
level=info msg="  --bpf-lb-dsr-dispatch='opt'" subsys=daemon
level=info msg="  --bpf-lb-dsr-l4-xlate='frontend'" subsys=daemon
level=info msg="  --bpf-lb-external-clusterip='false'" subsys=daemon
level=info msg="  --bpf-lb-maglev-hash-seed='JLfvgnHc2kaSUFaI'" subsys=daemon
level=info msg="  --bpf-lb-maglev-map-max='0'" subsys=daemon
level=info msg="  --bpf-lb-maglev-table-size='16381'" subsys=daemon
level=info msg="  --bpf-lb-map-max='65536'" subsys=daemon
level=info msg="  --bpf-lb-mode='snat'" subsys=daemon
level=info msg="  --bpf-lb-rev-nat-map-max='0'" subsys=daemon
level=info msg="  --bpf-lb-rss-ipv4-src-cidr=''" subsys=daemon
level=info msg="  --bpf-lb-rss-ipv6-src-cidr=''" subsys=daemon
level=info msg="  --bpf-lb-service-backend-map-max='0'" subsys=daemon
level=info msg="  --bpf-lb-service-map-max='0'" subsys=daemon
level=info msg="  --bpf-lb-sock='false'" subsys=daemon
level=info msg="  --bpf-lb-sock-hostns-only='false'" subsys=daemon
level=info msg="  --bpf-lb-source-range-map-max='0'" subsys=daemon
level=info msg="  --bpf-map-dynamic-size-ratio='0.0025'" subsys=daemon
level=info msg="  --bpf-nat-global-max='524288'" subsys=daemon
level=info msg="  --bpf-neigh-global-max='524288'" subsys=daemon
level=info msg="  --bpf-policy-map-max='16384'" subsys=daemon
level=info msg="  --bpf-root='/sys/fs/bpf'" subsys=daemon
level=info msg="  --bpf-sock-rev-map-max='262144'" subsys=daemon
level=info msg="  --bypass-ip-availability-upon-restore='false'" subsys=daemon
level=info msg="  --certificates-directory='/var/run/cilium/certs'" subsys=daemon
level=info msg="  --cflags=''" subsys=daemon
level=info msg="  --cgroup-root='/run/cilium/cgroupv2'" subsys=daemon
level=info msg="  --cluster-health-port='4240'" subsys=daemon
level=info msg="  --cluster-id='0'" subsys=daemon
level=info msg="  --cluster-name='default'" subsys=daemon
level=info msg="  --clustermesh-config='/var/lib/cilium/clustermesh/'" subsys=daemon
level=info msg="  --cmdref=''" subsys=daemon
level=info msg="  --cni-chaining-mode=''" subsys=daemon
level=info msg="  --config=''" subsys=daemon
level=info msg="  --config-dir='/tmp/cilium/config-map'" subsys=daemon
level=info msg="  --conntrack-gc-interval='0s'" subsys=daemon
level=info msg="  --crd-wait-timeout='5m0s'" subsys=daemon
level=info msg="  --datapath-mode='veth'" subsys=daemon
level=info msg="  --debug='false'" subsys=daemon
level=info msg="  --debug-verbose=''" subsys=daemon
level=info msg="  --derive-masquerade-ip-addr-from-device=''" subsys=daemon
level=info msg="  --devices=''" subsys=daemon
level=info msg="  --direct-routing-device=''" subsys=daemon
level=info msg="  --disable-cnp-status-updates='true'" subsys=daemon
level=info msg="  --disable-conntrack='false'" subsys=daemon
level=info msg="  --disable-endpoint-crd='false'" subsys=daemon
level=info msg="  --disable-envoy-version-check='false'" subsys=daemon
level=info msg="  --disable-iptables-feeder-rules=''" subsys=daemon
level=info msg="  --dns-max-ips-per-restored-rule='1000'" subsys=daemon
level=info msg="  --dns-policy-unload-on-shutdown='false'" subsys=daemon
level=info msg="  --dnsproxy-concurrency-limit='0'" subsys=daemon
level=info msg="  --dnsproxy-concurrency-processing-grace-period='0s'" subsys=daemon
level=info msg="  --egress-masquerade-interfaces=''" subsys=daemon
level=info msg="  --egress-multi-home-ip-rule-compat='false'" subsys=daemon
level=info msg="  --enable-auto-protect-node-port-range='true'" subsys=daemon
level=info msg="  --enable-bandwidth-manager='false'" subsys=daemon
level=info msg="  --enable-bbr='false'" subsys=daemon
level=info msg="  --enable-bgp-control-plane='false'" subsys=daemon
level=info msg="  --enable-bpf-clock-probe='true'" subsys=daemon
level=info msg="  --enable-bpf-masquerade='true'" subsys=daemon
level=info msg="  --enable-bpf-tproxy='false'" subsys=daemon
level=info msg="  --enable-cilium-endpoint-slice='false'" subsys=daemon
level=info msg="  --enable-custom-calls='false'" subsys=daemon
level=info msg="  --enable-endpoint-health-checking='true'" subsys=daemon
level=info msg="  --enable-endpoint-routes='false'" subsys=daemon
level=info msg="  --enable-envoy-config='false'" subsys=daemon
level=info msg="  --enable-external-ips='true'" subsys=daemon
level=info msg="  --enable-health-check-nodeport='true'" subsys=daemon
level=info msg="  --enable-health-checking='true'" subsys=daemon
level=info msg="  --enable-host-firewall='false'" subsys=daemon
level=info msg="  --enable-host-legacy-routing='false'" subsys=daemon
level=info msg="  --enable-host-port='true'" subsys=daemon
level=info msg="  --enable-host-reachable-services='false'" subsys=daemon
level=info msg="  --enable-hubble='true'" subsys=daemon
level=info msg="  --enable-hubble-recorder-api='true'" subsys=daemon
level=info msg="  --enable-icmp-rules='true'" subsys=daemon
level=info msg="  --enable-identity-mark='true'" subsys=daemon
level=info msg="  --enable-ip-masq-agent='false'" subsys=daemon
level=info msg="  --enable-ipsec='false'" subsys=daemon
level=info msg="  --enable-ipv4='true'" subsys=daemon
level=info msg="  --enable-ipv4-egress-gateway='true'" subsys=daemon
level=info msg="  --enable-ipv4-fragment-tracking='true'" subsys=daemon
level=info msg="  --enable-ipv4-masquerade='true'" subsys=daemon
level=info msg="  --enable-ipv6='false'" subsys=daemon
level=info msg="  --enable-ipv6-masquerade='true'" subsys=daemon
level=info msg="  --enable-ipv6-ndp='false'" subsys=daemon
level=info msg="  --enable-k8s-api-discovery='false'" subsys=daemon
level=info msg="  --enable-k8s-endpoint-slice='true'" subsys=daemon
level=info msg="  --enable-k8s-event-handover='false'" subsys=daemon
level=info msg="  --enable-k8s-terminating-endpoint='true'" subsys=daemon
level=info msg="  --enable-l2-neigh-discovery='true'" subsys=daemon
level=info msg="  --enable-l7-proxy='true'" subsys=daemon
level=info msg="  --enable-local-node-route='true'" subsys=daemon
level=info msg="  --enable-local-redirect-policy='false'" subsys=daemon
level=info msg="  --enable-mke='false'" subsys=daemon
level=info msg="  --enable-monitor='true'" subsys=daemon
level=info msg="  --enable-node-port='false'" subsys=daemon
level=info msg="  --enable-policy='default'" subsys=daemon
level=info msg="  --enable-recorder='false'" subsys=daemon
level=info msg="  --enable-remote-node-identity='true'" subsys=daemon
level=info msg="  --enable-runtime-device-detection='false'" subsys=daemon
level=info msg="  --enable-selective-regeneration='true'" subsys=daemon
level=info msg="  --enable-service-topology='false'" subsys=daemon
level=info msg="  --enable-session-affinity='false'" subsys=daemon
level=info msg="  --enable-svc-source-range-check='true'" subsys=daemon
level=info msg="  --enable-tracing='false'" subsys=daemon
level=info msg="  --enable-unreachable-routes='false'" subsys=daemon
level=info msg="  --enable-vtep='false'" subsys=daemon
level=info msg="  --enable-well-known-identities='false'" subsys=daemon
level=info msg="  --enable-wireguard='false'" subsys=daemon
level=info msg="  --enable-wireguard-userspace-fallback='false'" subsys=daemon
level=info msg="  --enable-xdp-prefilter='false'" subsys=daemon
level=info msg="  --enable-xt-socket-fallback='true'" subsys=daemon
level=info msg="  --encrypt-interface=''" subsys=daemon
level=info msg="  --encrypt-node='false'" subsys=daemon
level=info msg="  --endpoint-gc-interval='5m0s'" subsys=daemon
level=info msg="  --endpoint-interface-name-prefix=''" subsys=daemon
level=info msg="  --endpoint-queue-size='25'" subsys=daemon
level=info msg="  --endpoint-status=''" subsys=daemon
level=info msg="  --envoy-config-timeout='2m0s'" subsys=daemon
level=info msg="  --envoy-log=''" subsys=daemon
level=info msg="  --exclude-local-address=''" subsys=daemon
level=info msg="  --fixed-identity-mapping=''" subsys=daemon
level=info msg="  --force-local-policy-eval-at-source='true'" subsys=daemon
level=info msg="  --fqdn-regex-compile-lru-size='1024'" subsys=daemon
level=info msg="  --gops-port='9890'" subsys=daemon
level=info msg="  --host-reachable-services-protos='tcp,udp'" subsys=daemon
level=info msg="  --http-403-msg=''" subsys=daemon
level=info msg="  --http-idle-timeout='0'" subsys=daemon
level=info msg="  --http-max-grpc-timeout='0'" subsys=daemon
level=info msg="  --http-normalize-path='true'" subsys=daemon
level=info msg="  --http-request-timeout='3600'" subsys=daemon
level=info msg="  --http-retry-count='3'" subsys=daemon
level=info msg="  --http-retry-timeout='0'" subsys=daemon
level=info msg="  --hubble-disable-tls='false'" subsys=daemon
level=info msg="  --hubble-event-buffer-capacity='4095'" subsys=daemon
level=info msg="  --hubble-event-queue-size='0'" subsys=daemon
level=info msg="  --hubble-export-file-compress='false'" subsys=daemon
level=info msg="  --hubble-export-file-max-backups='5'" subsys=daemon
level=info msg="  --hubble-export-file-max-size-mb='10'" subsys=daemon
level=info msg="  --hubble-export-file-path=''" subsys=daemon
level=info msg="  --hubble-listen-address=':4244'" subsys=daemon
level=info msg="  --hubble-metrics=''" subsys=daemon
level=info msg="  --hubble-metrics-server=''" subsys=daemon
level=info msg="  --hubble-recorder-sink-queue-size='1024'" subsys=daemon
level=info msg="  --hubble-recorder-storage-path='/var/run/cilium/pcaps'" subsys=daemon
level=info msg="  --hubble-socket-path='/var/run/cilium/hubble.sock'" subsys=daemon
level=info msg="  --hubble-tls-cert-file='/var/lib/cilium/tls/hubble/server.crt'" subsys=daemon
level=info msg="  --hubble-tls-client-ca-files='/var/lib/cilium/tls/hubble/client-ca.crt'" subsys=daemon
level=info msg="  --hubble-tls-key-file='/var/lib/cilium/tls/hubble/server.key'" subsys=daemon
level=info msg="  --identity-allocation-mode='crd'" subsys=daemon
level=info msg="  --identity-change-grace-period='5s'" subsys=daemon
level=info msg="  --identity-restore-grace-period='10m0s'" subsys=daemon
level=info msg="  --install-egress-gateway-routes='false'" subsys=daemon
level=info msg="  --install-iptables-rules='true'" subsys=daemon
level=info msg="  --install-no-conntrack-iptables-rules='false'" subsys=daemon
level=info msg="  --ip-allocation-timeout='2m0s'" subsys=daemon
level=info msg="  --ip-masq-agent-config-path='/etc/config/ip-masq-agent'" subsys=daemon
level=info msg="  --ipam='cluster-pool'" subsys=daemon
level=info msg="  --ipsec-key-file=''" subsys=daemon
level=info msg="  --iptables-lock-timeout='5s'" subsys=daemon
level=info msg="  --iptables-random-fully='false'" subsys=daemon
level=info msg="  --ipv4-native-routing-cidr=''" subsys=daemon
level=info msg="  --ipv4-node='auto'" subsys=daemon
level=info msg="  --ipv4-pod-subnets=''" subsys=daemon
level=info msg="  --ipv4-range='auto'" subsys=daemon
level=info msg="  --ipv4-service-loopback-address='169.254.42.1'" subsys=daemon
level=info msg="  --ipv4-service-range='auto'" subsys=daemon
level=info msg="  --ipv6-cluster-alloc-cidr='f00d::/64'" subsys=daemon
level=info msg="  --ipv6-mcast-device=''" subsys=daemon
level=info msg="  --ipv6-native-routing-cidr=''" subsys=daemon
level=info msg="  --ipv6-node='auto'" subsys=daemon
level=info msg="  --ipv6-pod-subnets=''" subsys=daemon
level=info msg="  --ipv6-range='auto'" subsys=daemon
level=info msg="  --ipv6-service-range='auto'" subsys=daemon
level=info msg="  --join-cluster='false'" subsys=daemon
level=info msg="  --k8s-api-server=''" subsys=daemon
level=info msg="  --k8s-heartbeat-timeout='30s'" subsys=daemon
level=info msg="  --k8s-kubeconfig-path=''" subsys=daemon
level=info msg="  --k8s-namespace='kube-system'" subsys=daemon
level=info msg="  --k8s-require-ipv4-pod-cidr='false'" subsys=daemon
level=info msg="  --k8s-require-ipv6-pod-cidr='false'" subsys=daemon
level=info msg="  --k8s-service-cache-size='128'" subsys=daemon
level=info msg="  --k8s-service-proxy-name=''" subsys=daemon
level=info msg="  --k8s-sync-timeout='3m0s'" subsys=daemon
level=info msg="  --k8s-watcher-endpoint-selector='metadata.name!=kube-scheduler,metadata.name!=kube-controller-manager,metadata.name!=etcd-operator,metadata.name!=gcp-controller-manager'" subsys=daemon
level=info msg="  --keep-config='false'" subsys=daemon
level=info msg="  --kube-proxy-replacement='strict'" subsys=daemon
level=info msg="  --kube-proxy-replacement-healthz-bind-address=''" subsys=daemon
level=info msg="  --kvstore=''" subsys=daemon
level=info msg="  --kvstore-connectivity-timeout='2m0s'" subsys=daemon
level=info msg="  --kvstore-lease-ttl='15m0s'" subsys=daemon
level=info msg="  --kvstore-max-consecutive-quorum-errors='2'" subsys=daemon
level=info msg="  --kvstore-opt=''" subsys=daemon
level=info msg="  --kvstore-periodic-sync='5m0s'" subsys=daemon
level=info msg="  --label-prefix-file=''" subsys=daemon
level=info msg="  --labels=''" subsys=daemon
level=info msg="  --lib-dir='/var/lib/cilium'" subsys=daemon
level=info msg="  --local-max-addr-scope='252'" subsys=daemon
level=info msg="  --local-router-ipv4=''" subsys=daemon
level=info msg="  --local-router-ipv6=''" subsys=daemon
level=info msg="  --log-driver=''" subsys=daemon
level=info msg="  --log-opt=''" subsys=daemon
level=info msg="  --log-system-load='false'" subsys=daemon
level=info msg="  --max-controller-interval='0'" subsys=daemon
level=info msg="  --metrics=''" subsys=daemon
level=info msg="  --mke-cgroup-mount=''" subsys=daemon
level=info msg="  --monitor-aggregation='medium'" subsys=daemon
level=info msg="  --monitor-aggregation-flags='all'" subsys=daemon
level=info msg="  --monitor-aggregation-interval='5s'" subsys=daemon
level=info msg="  --monitor-queue-size='0'" subsys=daemon
level=info msg="  --mtu='0'" subsys=daemon
level=info msg="  --node-port-acceleration='disabled'" subsys=daemon
level=info msg="  --node-port-algorithm='random'" subsys=daemon
level=info msg="  --node-port-bind-protection='true'" subsys=daemon
level=info msg="  --node-port-mode='snat'" subsys=daemon
level=info msg="  --node-port-range='30000,32767'" subsys=daemon
level=info msg="  --policy-audit-mode='false'" subsys=daemon
level=info msg="  --policy-queue-size='100'" subsys=daemon
level=info msg="  --policy-trigger-interval='1s'" subsys=daemon
level=info msg="  --pprof='false'" subsys=daemon
level=info msg="  --pprof-port='6060'" subsys=daemon
level=info msg="  --preallocate-bpf-maps='false'" subsys=daemon
level=info msg="  --prepend-iptables-chains='true'" subsys=daemon
level=info msg="  --procfs='/host/proc'" subsys=daemon
level=info msg="  --prometheus-serve-addr=':9962'" subsys=daemon
level=info msg="  --proxy-connect-timeout='1'" subsys=daemon
level=info msg="  --proxy-gid='1337'" subsys=daemon
level=info msg="  --proxy-max-connection-duration-seconds='0'" subsys=daemon
level=info msg="  --proxy-max-requests-per-connection='0'" subsys=daemon
level=info msg="  --proxy-prometheus-port='9964'" subsys=daemon
level=info msg="  --read-cni-conf=''" subsys=daemon
level=info msg="  --restore='true'" subsys=daemon
level=info msg="  --route-metric='0'" subsys=daemon
level=info msg="  --sidecar-istio-proxy-image='cilium/istio_proxy'" subsys=daemon
level=info msg="  --single-cluster-route='false'" subsys=daemon
level=info msg="  --socket-path='/var/run/cilium/cilium.sock'" subsys=daemon
level=info msg="  --sockops-enable='false'" subsys=daemon
level=info msg="  --state-dir='/var/run/cilium'" subsys=daemon
level=info msg="  --tofqdns-dns-reject-response-code='refused'" subsys=daemon
level=info msg="  --tofqdns-enable-dns-compression='true'" subsys=daemon
level=info msg="  --tofqdns-endpoint-max-ip-per-hostname='50'" subsys=daemon
level=info msg="  --tofqdns-idle-connection-grace-period='0s'" subsys=daemon
level=info msg="  --tofqdns-max-deferred-connection-deletes='10000'" subsys=daemon
level=info msg="  --tofqdns-min-ttl='3600'" subsys=daemon
level=info msg="  --tofqdns-pre-cache=''" subsys=daemon
level=info msg="  --tofqdns-proxy-port='0'" subsys=daemon
level=info msg="  --tofqdns-proxy-response-max-delay='100ms'" subsys=daemon
level=info msg="  --trace-payloadlen='128'" subsys=daemon
level=info msg="  --tunnel='vxlan'" subsys=daemon
level=info msg="  --tunnel-port='0'" subsys=daemon
level=info msg="  --version='false'" subsys=daemon
level=info msg="  --vlan-bpf-bypass=''" subsys=daemon
level=info msg="  --vtep-cidr=''" subsys=daemon
level=info msg="  --vtep-endpoint=''" subsys=daemon
level=info msg="  --vtep-mac=''" subsys=daemon
level=info msg="  --vtep-mask=''" subsys=daemon
level=info msg="  --write-cni-conf-when-ready=''" subsys=daemon
level=info msg="     _ _ _" subsys=daemon
level=info msg=" ___|_| |_|_ _ _____" subsys=daemon
level=info msg="|  _| | | | | |     |" subsys=daemon
level=info msg="|___|_|_|_|___|_|_|_|" subsys=daemon
level=info msg="Cilium 1.12.3 1c466d2 2022-10-12T11:33:37+01:00 go version go1.18.6 linux/amd64" subsys=daemon
level=info msg="cilium-envoy  version: 48cbb2f88edca93ba80d1135dbb067a827b8adc2/1.21.5/Distribution/RELEASE/BoringSSL" subsys=daemon
level=info msg="clang (10.0.0) and kernel (5.4.17) versions: OK!" subsys=linux-datapath
level=info msg="linking environment: OK!" subsys=linux-datapath
level=info msg="Detected mounted BPF filesystem at /sys/fs/bpf" subsys=bpf
level=info msg="Mounted cgroupv2 filesystem at /run/cilium/cgroupv2" subsys=cgroups
level=info msg="Parsing base label prefixes from default label list" subsys=labels-filter
level=info msg="Parsing additional label prefixes from user inputs: []" subsys=labels-filter
level=info msg="Final label prefixes to be used for identity evaluation:" subsys=labels-filter
level=info msg=" - reserved:.*" subsys=labels-filter
level=info msg=" - :io\\.kubernetes\\.pod\\.namespace" subsys=labels-filter
level=info msg=" - :io\\.cilium\\.k8s\\.namespace\\.labels" subsys=labels-filter
level=info msg=" - :app\\.kubernetes\\.io" subsys=labels-filter
level=info msg=" - !:io\\.kubernetes" subsys=labels-filter
level=info msg=" - !:kubernetes\\.io" subsys=labels-filter
level=info msg=" - !:.*beta\\.kubernetes\\.io" subsys=labels-filter
level=info msg=" - !:k8s\\.io" subsys=labels-filter
level=info msg=" - !:pod-template-generation" subsys=labels-filter
level=info msg=" - !:pod-template-hash" subsys=labels-filter
level=info msg=" - !:controller-revision-hash" subsys=labels-filter
level=info msg=" - !:annotation.*" subsys=labels-filter
level=info msg=" - !:etcd_node" subsys=labels-filter
level=info msg="Auto-disabling \"enable-bpf-clock-probe\" feature since KERNEL_HZ cannot be determined" error="Cannot probe CONFIG_HZ" subsys=daemon
level=info msg="Using autogenerated IPv4 allocation range" subsys=node v4Prefix=10.32.0.0/16
level=info msg="Initializing daemon" subsys=daemon
level=info msg="Establishing connection to apiserver" host="https://10.233.0.1:443" subsys=k8s
level=info msg="Connected to apiserver" subsys=k8s
level=info msg="Trying to auto-enable \"enable-node-port\", \"enable-external-ips\", \"bpf-lb-sock\", \"enable-host-port\", \"enable-session-affinity\" features" subsys=daemon
level=warning msg="Session affinity for host reachable services needs kernel 5.7.0 or newer to work properly when accessed from inside cluster: the same service endpoint will be selected from all network namespaces on the host." subsys=daemon
level=info msg="Inheriting MTU from external network interface" device=bond0.10 ipAddr=172.16.10.32 mtu=1500 subsys=mtu
level=info msg="Envoy: Starting xDS gRPC server listening on /var/run/cilium/xds.sock" subsys=envoy-manager
level=info msg="Restored backends from maps" failedBackends=0 restoredBackends=107 subsys=service
level=info msg="Restored services from maps" failedServices=0 restoredServices=71 subsys=service
level=info msg="Reading old endpoints..." subsys=daemon
level=info msg="No old endpoints found." subsys=daemon
level=info msg="Waiting until all Cilium CRDs are available" subsys=k8s
level=info msg="All Cilium CRDs have been found and are available" subsys=k8s
level=info msg="Creating or updating CiliumNode resource" node=prod-ceph-02 subsys=nodediscovery
level=info msg="Retrieved node information from cilium node" nodeName=prod-ceph-02 subsys=k8s
level=info msg="Received own node information from API server" ipAddr.ipv4=172.16.10.32 ipAddr.ipv6="<nil>" k8sNodeIP=172.16.10.32 labels="map[beta.kubernetes.io/arch:amd64 beta.kubernetes.io/os:linux kubernetes.io/arch:amd64 kubernetes.io/hostname:prod-ceph-02 kubernetes.io/os:linux node-role.kubernetes.io/worker:]" nodeName=prod-ceph-02 subsys=k8s v4Prefix=10.0.9.0/24 v6Prefix="<nil>"
level=info msg="k8s mode: Allowing localhost to reach local endpoints" subsys=daemon
level=info msg="Detected devices" devices="[bond0.10]" subsys=linux-datapath
level=info msg="BPF host routing requires kernel 5.10 or newer. Falling back to legacy host routing (enable-host-legacy-routing=true)." subsys=daemon
level=warning msg="both egress gateway and L7 proxy (--enable-l7-proxy) are enabled. This is currently not fully supported: if the same endpoint is selected both by an egress gateway and a L7 policy, endpoint traffic will not go through egress gateway." subsys=daemon url="https://github.com/cilium/cilium/issues/19642"
level=info msg="Masquerading IP selected for device" device=bond0.10 ipv4=172.16.10.32 subsys=node
level=info msg="Enabling k8s event listener" subsys=k8s-watcher
level=info msg="Policy Add Request" ciliumNetworkPolicy="[&{EndpointSelector:{\"matchLabels\":{\"k8s:app.kubernetes.io/name\":\"argocd-application-controller\",\"k8s:io.kubernetes.pod.namespace\":\"argocd\"}} NodeSelector:{} Ingress:[{IngressCommonRule:{FromEndpoints:[{\"matchExpressions\":[{\"key\":\"k8s:io.kubernetes.pod.namespace\",\"operator\":\"Exists\"}]}] FromRequires:[] FromCIDR: FromCIDRSet:[] FromEntities:[] aggregatedSelectors:[]} ToPorts:[{Ports:[{Port:8082 Protocol:TCP}] TerminatingTLS:<nil> OriginatingTLS:<nil> Rules:<nil>}] ICMPs:[]}] IngressDeny:[] Egress:[] EgressDeny:[] Labels:[k8s:io.cilium.k8s.policy.derived-from=NetworkPolicy k8s:io.cilium.k8s.policy.name=argocd-application-controller-network-policy k8s:io.cilium.k8s.policy.namespace=argocd k8s:io.cilium.k8s.policy.uid=2fec798f-4844-471d-9f91-9b5c40651bea] Description:}]" policyAddRequest=a2d6de93-2d5b-4af7-9b74-73e0c48fbf0c subsys=daemon
level=info msg="Policy imported via API, recalculating..." policyAddRequest=a2d6de93-2d5b-4af7-9b74-73e0c48fbf0c policyRevision=2 subsys=daemon
level=info msg="NetworkPolicy successfully added" k8sApiVersion= k8sNetworkPolicyName=argocd-application-controller-network-policy subsys=k8s-watcher
level=info msg="Policy Add Request" ciliumNetworkPolicy="[&{EndpointSelector:{\"matchLabels\":{\"k8s:app.kubernetes.io/name\":\"argocd-dex-server\",\"k8s:io.kubernetes.pod.namespace\":\"argocd\"}} NodeSelector:{} Ingress:[{IngressCommonRule:{FromEndpoints:[{\"matchLabels\":{\"k8s:app.kubernetes.io/name\":\"argocd-server\",\"k8s:io.kubernetes.pod.namespace\":\"argocd\"}}] FromRequires:[] FromCIDR: FromCIDRSet:[] FromEntities:[] aggregatedSelectors:[]} ToPorts:[{Ports:[{Port:5556 Protocol:TCP}] TerminatingTLS:<nil> OriginatingTLS:<nil> Rules:<nil>} {Ports:[{Port:5557 Protocol:TCP}] TerminatingTLS:<nil> OriginatingTLS:<nil> Rules:<nil>}] ICMPs:[]} {IngressCommonRule:{FromEndpoints:[{\"matchExpressions\":[{\"key\":\"k8s:io.kubernetes.pod.namespace\",\"operator\":\"Exists\"}]}] FromRequires:[] FromCIDR: FromCIDRSet:[] FromEntities:[] aggregatedSelectors:[]} ToPorts:[{Ports:[{Port:5558 Protocol:TCP}] TerminatingTLS:<nil> OriginatingTLS:<nil> Rules:<nil>}] ICMPs:[]}] IngressDeny:[] Egress:[] EgressDeny:[] Labels:[k8s:io.cilium.k8s.policy.derived-from=NetworkPolicy k8s:io.cilium.k8s.policy.name=argocd-dex-server-network-policy k8s:io.cilium.k8s.policy.namespace=argocd k8s:io.cilium.k8s.policy.uid=166c8846-5ba8-486d-a119-4a7c0b6eb481] Description:}]" policyAddRequest=eb968389-e99c-40a6-b7e4-5e6433a4a071 subsys=daemon
level=info msg="Policy imported via API, recalculating..." policyAddRequest=eb968389-e99c-40a6-b7e4-5e6433a4a071 policyRevision=3 subsys=daemon
level=info msg="NetworkPolicy successfully added" k8sApiVersion= k8sNetworkPolicyName=argocd-dex-server-network-policy subsys=k8s-watcher
level=info msg="Waiting until local node addressing before starting watchers depending on it" subsys=k8s-watcher
level=info msg="Removing stale endpoint interfaces" subsys=daemon
level=info msg="Policy Add Request" ciliumNetworkPolicy="[&{EndpointSelector:{\"matchLabels\":{\"k8s:app.kubernetes.io/name\":\"argocd-redis-ha-haproxy\",\"k8s:io.kubernetes.pod.namespace\":\"argocd\"}} NodeSelector:{} Ingress:[{IngressCommonRule:{FromEndpoints:[{\"matchLabels\":{\"k8s:app.kubernetes.io/name\":\"argocd-server\",\"k8s:io.kubernetes.pod.namespace\":\"argocd\"}}] FromRequires:[] FromCIDR: FromCIDRSet:[] FromEntities:[] aggregatedSelectors:[]} ToPorts:[] ICMPs:[]} {IngressCommonRule:{FromEndpoints:[{\"matchLabels\":{\"k8s:app.kubernetes.io/name\":\"argocd-repo-server\",\"k8s:io.kubernetes.pod.namespace\":\"argocd\"}}] FromRequires:[] FromCIDR: FromCIDRSet:[] FromEntities:[] aggregatedSelectors:[]} ToPorts:[] ICMPs:[]} {IngressCommonRule:{FromEndpoints:[{\"matchLabels\":{\"k8s:app.kubernetes.io/name\":\"argocd-application-controller\",\"k8s:io.kubernetes.pod.namespace\":\"argocd\"}}] FromRequires:[] FromCIDR: FromCIDRSet:[] FromEntities:[] aggregatedSelectors:[]} ToPorts:[] ICMPs:[]} {IngressCommonRule:{FromEndpoints:[{\"matchLabels\":{\"k8s:app.kubernetes.io/name\":\"argocd-redis-ha\",\"k8s:io.kubernetes.pod.namespace\":\"argocd\"}}] FromRequires:[] FromCIDR: FromCIDRSet:[] FromEntities:[] aggregatedSelectors:[]} ToPorts:[] ICMPs:[]}] IngressDeny:[] Egress:[] EgressDeny:[] Labels:[k8s:io.cilium.k8s.policy.derived-from=NetworkPolicy k8s:io.cilium.k8s.policy.name=argocd-redis-ha-proxy-network-policy k8s:io.cilium.k8s.policy.namespace=argocd k8s:io.cilium.k8s.policy.uid=0a1f8737-0378-42e4-921e-b14a8d66b80d] Description:}]" policyAddRequest=668652af-ff51-421e-87c0-98e4057542cc subsys=daemon
level=info msg="Policy imported via API, recalculating..." policyAddRequest=668652af-ff51-421e-87c0-98e4057542cc policyRevision=4 subsys=daemon
level=info msg="NetworkPolicy successfully added" k8sApiVersion= k8sNetworkPolicyName=argocd-redis-ha-proxy-network-policy subsys=k8s-watcher
level=info msg="Policy Add Request" ciliumNetworkPolicy="[&{EndpointSelector:{\"matchLabels\":{\"k8s:app.kubernetes.io/name\":\"argocd-redis-ha\",\"k8s:io.kubernetes.pod.namespace\":\"argocd\"}} NodeSelector:{} Ingress:[{IngressCommonRule:{FromEndpoints:[{\"matchLabels\":{\"k8s:app.kubernetes.io/name\":\"argocd-redis-ha-haproxy\",\"k8s:io.kubernetes.pod.namespace\":\"argocd\"}}] FromRequires:[] FromCIDR: FromCIDRSet:[] FromEntities:[] aggregatedSelectors:[]} ToPorts:[] ICMPs:[]} {IngressCommonRule:{FromEndpoints:[{\"matchLabels\":{\"k8s:app.kubernetes.io/name\":\"argocd-redis-ha\",\"k8s:io.kubernetes.pod.namespace\":\"argocd\"}}] FromRequires:[] FromCIDR: FromCIDRSet:[] FromEntities:[] aggregatedSelectors:[]} ToPorts:[] ICMPs:[]}] IngressDeny:[] Egress:[] EgressDeny:[] Labels:[k8s:io.cilium.k8s.policy.derived-from=NetworkPolicy k8s:io.cilium.k8s.policy.name=argocd-redis-ha-server-network-policy k8s:io.cilium.k8s.policy.namespace=argocd k8s:io.cilium.k8s.policy.uid=11a3bb27-f15b-4535-985a-55fec281692a] Description:}]" policyAddRequest=80bd5b25-672d-4db0-a070-b230d2689afd subsys=daemon
level=info msg="Policy imported via API, recalculating..." policyAddRequest=80bd5b25-672d-4db0-a070-b230d2689afd policyRevision=5 subsys=daemon
level=info msg="NetworkPolicy successfully added" k8sApiVersion= k8sNetworkPolicyName=argocd-redis-ha-server-network-policy subsys=k8s-watcher
level=info msg="Policy Add Request" ciliumNetworkPolicy="[&{EndpointSelector:{\"matchLabels\":{\"k8s:app.kubernetes.io/name\":\"argocd-repo-server\",\"k8s:io.kubernetes.pod.namespace\":\"argocd\"}} NodeSelector:{} Ingress:[{IngressCommonRule:{FromEndpoints:[{\"matchLabels\":{\"k8s:app.kubernetes.io/name\":\"argocd-server\",\"k8s:io.kubernetes.pod.namespace\":\"argocd\"}}] FromRequires:[] FromCIDR: FromCIDRSet:[] FromEntities:[] aggregatedSelectors:[]} ToPorts:[{Ports:[{Port:8081 Protocol:TCP}] TerminatingTLS:<nil> OriginatingTLS:<nil> Rules:<nil>}] ICMPs:[]} {IngressCommonRule:{FromEndpoints:[{\"matchLabels\":{\"k8s:app.kubernetes.io/name\":\"argocd-application-controller\",\"k8s:io.kubernetes.pod.namespace\":\"argocd\"}}] FromRequires:[] FromCIDR: FromCIDRSet:[] FromEntities:[] aggregatedSelectors:[]} ToPorts:[{Ports:[{Port:8081 Protocol:TCP}] TerminatingTLS:<nil> OriginatingTLS:<nil> Rules:<nil>}] ICMPs:[]} {IngressCommonRule:{FromEndpoints:[{\"matchLabels\":{\"k8s:app.kubernetes.io/name\":\"argocd-notifications-controller\",\"k8s:io.kubernetes.pod.namespace\":\"argocd\"}}] FromRequires:[] FromCIDR: FromCIDRSet:[] FromEntities:[] aggregatedSelectors:[]} ToPorts:[{Ports:[{Port:8081 Protocol:TCP}] TerminatingTLS:<nil> OriginatingTLS:<nil> Rules:<nil>}] ICMPs:[]} {IngressCommonRule:{FromEndpoints:[{\"matchExpressions\":[{\"key\":\"k8s:io.kubernetes.pod.namespace\",\"operator\":\"Exists\"}]}] FromRequires:[] FromCIDR: FromCIDRSet:[] FromEntities:[] aggregatedSelectors:[]} ToPorts:[{Ports:[{Port:8084 Protocol:TCP}] TerminatingTLS:<nil> OriginatingTLS:<nil> Rules:<nil>}] ICMPs:[]}] IngressDeny:[] Egress:[] EgressDeny:[] Labels:[k8s:io.cilium.k8s.policy.derived-from=NetworkPolicy k8s:io.cilium.k8s.policy.name=argocd-repo-server-network-policy k8s:io.cilium.k8s.policy.namespace=argocd k8s:io.cilium.k8s.policy.uid=f1be1f1a-1a78-4b50-9390-6bade59ef095] Description:}]" policyAddRequest=68621620-1bc4-479e-bc82-0aeee1f96fad subsys=daemon
level=info msg="Policy imported via API, recalculating..." policyAddRequest=68621620-1bc4-479e-bc82-0aeee1f96fad policyRevision=6 subsys=daemon
level=info msg="NetworkPolicy successfully added" k8sApiVersion= k8sNetworkPolicyName=argocd-repo-server-network-policy subsys=k8s-watcher
level=warning msg="CiliumEgressNATPolicy is deprecated and will be removed in version 1.13. Use CiliumEgressGatewayPolicy instead." subsys=klog
level=info msg="Policy Add Request" ciliumNetworkPolicy="[&{EndpointSelector:{\"matchLabels\":{\"k8s:app.kubernetes.io/name\":\"argocd-server\",\"k8s:io.kubernetes.pod.namespace\":\"argocd\"}} NodeSelector:{} Ingress:[{IngressCommonRule:{FromEndpoints:[{}] FromRequires:[] FromCIDR: FromCIDRSet:[] FromEntities:[] aggregatedSelectors:[]} ToPorts:[] ICMPs:[]}] IngressDeny:[] Egress:[] EgressDeny:[] Labels:[k8s:io.cilium.k8s.policy.derived-from=NetworkPolicy k8s:io.cilium.k8s.policy.name=argocd-server-network-policy k8s:io.cilium.k8s.policy.namespace=argocd k8s:io.cilium.k8s.policy.uid=3c0ec1e9-4827-4f5a-93d4-d112786d2db0] Description:}]" policyAddRequest=0e61fc9a-eca1-46a8-b73e-e4faf0b63d63 subsys=daemon
level=info msg="Policy imported via API, recalculating..." policyAddRequest=0e61fc9a-eca1-46a8-b73e-e4faf0b63d63 policyRevision=7 subsys=daemon
level=info msg="NetworkPolicy successfully added" k8sApiVersion= k8sNetworkPolicyName=argocd-server-network-policy subsys=k8s-watcher
level=warning msg="CiliumEgressNATPolicy is deprecated and will be removed in version 1.13. Use CiliumEgressGatewayPolicy instead." subsys=klog
level=info msg="Skipping kvstore configuration" subsys=daemon
level=info msg="Restored router address from node_config" file=/var/run/cilium/state/globals/node_config.h ipv4=10.0.9.61 ipv6="<nil>" subsys=node
level=info msg="Initializing node addressing" subsys=daemon
level=info msg="Initializing cluster-pool IPAM" subsys=ipam v4Prefix=10.0.9.0/24 v6Prefix="<nil>"
level=info msg="Restoring endpoints..." subsys=daemon
level=info msg="Endpoints restored" failed=0 restored=0 subsys=daemon
level=info msg="Addressing information:" subsys=daemon
level=info msg="  Cluster-Name: default" subsys=daemon
level=info msg="  Cluster-ID: 0" subsys=daemon
level=info msg="  Local node-name: prod-ceph-02" subsys=daemon
level=info msg="  Node-IPv6: <nil>" subsys=daemon
level=info msg="  External-Node IPv4: 172.16.10.32" subsys=daemon
level=info msg="  Internal-Node IPv4: 10.0.9.61" subsys=daemon
level=info msg="  IPv4 allocation prefix: 10.0.9.0/24" subsys=daemon
level=info msg="  Loopback IPv4: 169.254.42.1" subsys=daemon
level=info msg="  Local IPv4 addresses:" subsys=daemon
level=info msg="  - 172.16.10.32" subsys=daemon
level=info msg="  - 10.0.9.61" subsys=daemon
level=info msg="Adding local node to cluster" node="{prod-ceph-02 default [{InternalIP 172.16.10.32} {CiliumInternalIP 10.0.9.61}] 10.0.9.0/24 [] <nil> [] 10.0.9.136 <nil> <nil> <nil> 0 local 0 map[beta.kubernetes.io/arch:amd64 beta.kubernetes.io/os:linux kubernetes.io/arch:amd64 kubernetes.io/hostname:prod-ceph-02 kubernetes.io/os:linux node-role.kubernetes.io/worker:] 1 }" subsys=nodediscovery
level=info msg="Creating or updating CiliumNode resource" node=prod-ceph-02 subsys=nodediscovery
level=info msg="Waiting until all pre-existing resources have been received" subsys=k8s-watcher
level=info msg="Initializing identity allocator" subsys=identity-cache
level=info msg="Allocating identities between range" cluster-id=0 max=65535 min=256 subsys=identity-cache
level=info msg="Cluster-ID is not specified, skipping ClusterMesh initialization" subsys=daemon
level=info msg="Setting up BPF datapath" bpfClockSource=ktime bpfInsnSet=v2 subsys=datapath-loader
level=info msg="Setting sysctl" subsys=sysctl sysParamName=net.core.bpf_jit_enable sysParamValue=1
level=info msg="Setting sysctl" subsys=sysctl sysParamName=net.ipv4.conf.all.rp_filter sysParamValue=0
level=info msg="Setting sysctl" subsys=sysctl sysParamName=net.ipv4.fib_multipath_use_neigh sysParamValue=1
level=info msg="Setting sysctl" subsys=sysctl sysParamName=kernel.unprivileged_bpf_disabled sysParamValue=1
level=info msg="Setting sysctl" subsys=sysctl sysParamName=kernel.timer_migration sysParamValue=0
level=info msg="Setting sysctl" subsys=sysctl sysParamName=net.ipv4.conf.cilium_host.forwarding sysParamValue=1
level=info msg="Setting sysctl" subsys=sysctl sysParamName=net.ipv4.conf.cilium_host.rp_filter sysParamValue=0
level=info msg="Setting sysctl" subsys=sysctl sysParamName=net.ipv4.conf.cilium_host.accept_local sysParamValue=1
level=info msg="Setting sysctl" subsys=sysctl sysParamName=net.ipv4.conf.cilium_host.send_redirects sysParamValue=0
level=info msg="Setting sysctl" subsys=sysctl sysParamName=net.ipv4.conf.cilium_net.forwarding sysParamValue=1
level=info msg="Setting sysctl" subsys=sysctl sysParamName=net.ipv4.conf.cilium_net.rp_filter sysParamValue=0
level=info msg="Setting sysctl" subsys=sysctl sysParamName=net.ipv4.conf.cilium_net.accept_local sysParamValue=1
level=info msg="Setting sysctl" subsys=sysctl sysParamName=net.ipv4.conf.cilium_net.send_redirects sysParamValue=0
level=info msg="regenerating all endpoints" reason="one or more identities created or deleted" subsys=endpoint-manager
level=info msg="Trace[1006933274]: \"DeltaFIFO Pop Process\" ID:kubesphere-logging-system/elasticsearch-logging-discovery-0,Depth:97,Reason:slow event handlers blocking the queue (21-Oct-2022 06:20:21.702) (total time: 101ms):" subsys=klog
level=info msg="Trace[1006933274]: [101.887898ms] [101.887898ms] END" subsys=klog
level=info msg="regenerating all endpoints" reason="one or more identities created or deleted" subsys=endpoint-manager
level=info msg="regenerating all endpoints" reason= subsys=endpoint-manager
level=error msg="Command execution failed" cmd="[/var/lib/cilium/bpf/init.sh /var/lib/cilium/bpf /var/run/cilium/state /host/proc/sys/net /sys/class/net 10.0.9.61 <nil> tunnel vxlan 8472 bond0.10 cilium_host cilium_net 1500 true true false /run/cilium/cgroupv2 /sys/fs/bpf true true v2 16 false true 1]" error="exit status 1" subsys=datapath-loader
level=warning msg="+ set -o pipefail" subsys=datapath-loader
level=warning msg="+ rm /var/run/cilium/state/encap.state" subsys=datapath-loader
level=warning msg="+ true" subsys=datapath-loader
level=warning msg="+ DIR=/run/cilium/state/globals" subsys=datapath-loader
level=warning msg="+ case \"${MODE}\" in" subsys=datapath-loader
level=warning msg="+ sed -i '/^#.*CILIUM_NET_MAC.*$/d' /var/run/cilium/state/globals/node_config.h" subsys=datapath-loader
level=warning msg="++ ip link show cilium_net" subsys=datapath-loader
level=warning msg="++ grep ether" subsys=datapath-loader
level=warning msg="++ awk '{print $2}'" subsys=datapath-loader
level=warning msg="+ CILIUM_NET_MAC=5a:f3:bb:fc:25:c0" subsys=datapath-loader
level=warning msg="++ mac2array 5a:f3:bb:fc:25:c0" subsys=datapath-loader
level=warning msg="++ echo '{0x5a,0xf3,0xbb,0xfc,0x25,0xc0}'" subsys=datapath-loader
level=warning msg="+ CILIUM_NET_MAC='{0x5a,0xf3,0xbb,0xfc,0x25,0xc0}'" subsys=datapath-loader
level=warning msg="+ sed -i '/^#.*CILIUM_NET_MAC.*$/d' /var/run/cilium/state/globals/node_config.h" subsys=datapath-loader
level=warning msg="+ echo '#ifndef CILIUM_NET_MAC'" subsys=datapath-loader
level=warning msg="+ echo '#define CILIUM_NET_MAC { .addr = {0x5a,0xf3,0xbb,0xfc,0x25,0xc0}}'" subsys=datapath-loader
level=warning msg="+ echo '#endif /* CILIUM_NET_MAC */'" subsys=datapath-loader
level=warning msg="+ sed -i '/^#.*HOST_IFINDEX.*$/d' /var/run/cilium/state/globals/node_config.h" subsys=datapath-loader
level=warning msg="++ cat /sys/class/net/cilium_net/ifindex" subsys=datapath-loader
level=warning msg="+ HOST_IDX=12" subsys=datapath-loader
level=warning msg="+ echo '#define HOST_IFINDEX 12'" subsys=datapath-loader
level=warning msg="+ sed -i '/^#.*HOST_IFINDEX_MAC.*$/d' /var/run/cilium/state/globals/node_config.h" subsys=datapath-loader
level=warning msg="++ ip link show cilium_host" subsys=datapath-loader
level=warning msg="++ grep ether" subsys=datapath-loader
level=warning msg="++ awk '{print $2}'" subsys=datapath-loader
level=warning msg="+ HOST_MAC=1a:79:5a:6f:f8:c7" subsys=datapath-loader
level=warning msg="++ mac2array 1a:79:5a:6f:f8:c7" subsys=datapath-loader
level=warning msg="++ echo '{0x1a,0x79,0x5a,0x6f,0xf8,0xc7}'" subsys=datapath-loader
level=warning msg="+ HOST_MAC='{0x1a,0x79,0x5a,0x6f,0xf8,0xc7}'" subsys=datapath-loader
level=warning msg="+ echo '#define HOST_IFINDEX_MAC { .addr = {0x1a,0x79,0x5a,0x6f,0xf8,0xc7}}'" subsys=datapath-loader
level=warning msg="+ sed -i '/^#.*CILIUM_IFINDEX.*$/d' /var/run/cilium/state/globals/node_config.h" subsys=datapath-loader
level=warning msg="++ cat /sys/class/net/cilium_host/ifindex" subsys=datapath-loader
level=warning msg="+ CILIUM_IDX=13" subsys=datapath-loader
level=warning msg="+ echo '#define CILIUM_IFINDEX 13'" subsys=datapath-loader
level=warning msg="++ cat /host/proc/sys/net/ipv4/ip_local_port_range" subsys=datapath-loader
level=warning msg="++ awk '{print $1}'" subsys=datapath-loader
level=warning msg="+ CILIUM_EPHEMERAL_MIN=32768" subsys=datapath-loader
level=warning msg="+ echo '#define EPHEMERAL_MIN 32768'" subsys=datapath-loader
level=warning msg="+ '[' '<nil>' '!=' '<nil>' ']'" subsys=datapath-loader
level=warning msg="+ '[' 10.0.9.61 '!=' '<nil>' ']'" subsys=datapath-loader
level=warning msg="++ ip -4 addr show to 10.0.9.61 dev cilium_host" subsys=datapath-loader
level=warning msg="+ '[' -n '13: cilium_host@cilium_net: <BROADCAST,MULTICAST,NOARP,UP,LOWER_UP> mtu 1500 qdisc noqueue state UP group default qlen 1000" subsys=datapath-loader
level=warning msg="    inet 10.0.9.61/32 scope link cilium_host" subsys=datapath-loader
level=warning msg="       valid_lft forever preferred_lft forever' ']'" subsys=datapath-loader
level=warning msg="+ '[' true = true ']'" subsys=datapath-loader
level=warning msg="+ move_local_rules" subsys=datapath-loader
level=warning msg="+ '[' 10.0.9.61 '!=' '<nil>' ']'" subsys=datapath-loader
level=warning msg="+ move_local_rules_af 'ip -4'" subsys=datapath-loader
level=warning msg="+ IP='ip -4'" subsys=datapath-loader
level=warning msg="++ ip -4 rule list" subsys=datapath-loader
level=warning msg="+ '[' -z '9:\tfrom all fwmark 0x200/0xf00 lookup 2004" subsys=datapath-loader
level=warning msg="10:\tfrom all fwmark 0xa00/0xf00 lookup 2005" subsys=datapath-loader
level=warning msg="100:\tfrom all lookup local" subsys=datapath-loader
level=warning msg="32766:\tfrom all lookup main" subsys=datapath-loader
level=warning msg="32767:\tfrom all lookup default' ']'" subsys=datapath-loader
level=warning msg="+ ip -4 rule list" subsys=datapath-loader
level=warning msg="+ grep 100" subsys=datapath-loader
level=warning msg="+ grep 'lookup local'" subsys=datapath-loader
level=warning msg="100:\tfrom all lookup local" subsys=datapath-loader
level=warning msg="+ ip -4 rule del from all lookup local pref 0" subsys=datapath-loader
level=warning msg="+ true" subsys=datapath-loader
level=warning msg="++ ip -4 rule list" subsys=datapath-loader
level=warning msg="++ grep 'lookup local'" subsys=datapath-loader
level=warning msg="++ wc -l" subsys=datapath-loader
level=warning msg="+ '[' 1 -eq 0 ']'" subsys=datapath-loader
level=warning msg="+ '[' '<nil>' '!=' '<nil>' ']'" subsys=datapath-loader
level=warning msg="+ setup_proxy_rules" subsys=datapath-loader
level=warning msg="+ from_ingress_rulespec='fwmark 0xA00/0xF00 pref 10 lookup 2005'" subsys=datapath-loader
level=warning msg="+ to_proxy_rulespec='fwmark 0x200/0xF00 pref 9 lookup 2004'" subsys=datapath-loader
level=warning msg="+ '[' 10.0.9.61 '!=' '<nil>' ']'" subsys=datapath-loader
level=warning msg="++ ip -4 rule list" subsys=datapath-loader
level=warning msg="+ '[' -n '9:\tfrom all fwmark 0x200/0xf00 lookup 2004" subsys=datapath-loader
level=warning msg="10:\tfrom all fwmark 0xa00/0xf00 lookup 2005" subsys=datapath-loader
level=warning msg="100:\tfrom all lookup local" subsys=datapath-loader
level=warning msg="32766:\tfrom all lookup main" subsys=datapath-loader
level=warning msg="32767:\tfrom all lookup default' ']'" subsys=datapath-loader
level=warning msg="++ ip -4 rule list fwmark 0x200/0xF00 pref 9 lookup 2004" subsys=datapath-loader
level=warning msg="+ '[' -z '9:\tfrom all fwmark 0x200/0xf00 lookup 2004' ']'" subsys=datapath-loader
level=warning msg="+ '[' false = true ']'" subsys=datapath-loader
level=warning msg="++ ip -4 rule list fwmark 0xA00/0xF00 pref 10 lookup 2005" subsys=datapath-loader
level=warning msg="+ '[' -z '10:\tfrom all fwmark 0xa00/0xf00 lookup 2005' ']'" subsys=datapath-loader
level=warning msg="+ ip route replace table 2004 local 0.0.0.0/0 dev lo" subsys=datapath-loader
level=warning msg="+ '[' false = true ']'" subsys=datapath-loader
level=warning msg="+ ip route replace table 2005 10.0.9.61/32 dev cilium_host" subsys=datapath-loader
level=warning msg="+ ip route replace table 2005 default via 10.0.9.61" subsys=datapath-loader
level=warning msg="+ '[' '<nil>' '!=' '<nil>' ']'" subsys=datapath-loader
level=warning msg="+ ip -6 rule del fwmark 0x200/0xF00 pref 9 lookup 2004" subsys=datapath-loader
level=warning msg="+ true" subsys=datapath-loader
level=warning msg="+ ip -6 rule del fwmark 0xA00/0xF00 pref 10 lookup 2005" subsys=datapath-loader
level=warning msg="+ true" subsys=datapath-loader
level=warning msg="+ '[' tunnel = ipip ']'" subsys=datapath-loader
level=warning msg="+ ip link del cilium_ipip4" subsys=datapath-loader
level=warning msg="+ true" subsys=datapath-loader
level=warning msg="+ ip link del cilium_ipip6" subsys=datapath-loader
level=warning msg="+ true" subsys=datapath-loader
level=warning msg="+ ip link del cilium_sit" subsys=datapath-loader
level=warning msg="+ true" subsys=datapath-loader
level=warning msg="+ '[' tunnel = tunnel ']'" subsys=datapath-loader
level=warning msg="+ sed -i '/^#.*TUNNEL_MODE.*$/d' /var/run/cilium/state/globals/node_config.h" subsys=datapath-loader
level=warning msg="+ echo '#define TUNNEL_MODE 1'" subsys=datapath-loader
level=warning msg="+ '[' vxlan '!=' '<nil>' ']'" subsys=datapath-loader
level=warning msg="+ ENCAP_DEV=cilium_vxlan" subsys=datapath-loader
level=warning msg="+ ip link show cilium_vxlan" subsys=datapath-loader
level=warning msg="14: cilium_vxlan: <BROADCAST,MULTICAST,UP,LOWER_UP> mtu 1500 qdisc noqueue state UNKNOWN mode DEFAULT group default qlen 1000" subsys=datapath-loader
level=warning msg="    link/ether e6:1e:5f:e4:16:3c brd ff:ff:ff:ff:ff:ff" subsys=datapath-loader
level=warning msg="+ '[' 8472 '!=' '<nil>' ']'" subsys=datapath-loader
level=warning msg="+ ip -details link show cilium_vxlan" subsys=datapath-loader
level=warning msg="+ grep 'dstport 8472'" subsys=datapath-loader
level=warning msg="    vxlan external id 0 srcport 0 0 dstport 8472 nolearning ttl auto ageing 300 udpcsum noudp6zerocsumtx noudp6zerocsumrx addrgenmode eui64 numtxqueues 1 numrxqueues 1 gso_max_size 65536 gso_max_segs 65535 " subsys=datapath-loader
level=warning msg="+ ip link set cilium_vxlan mtu 1500" subsys=datapath-loader
level=warning msg="+ setup_dev cilium_vxlan" subsys=datapath-loader
level=warning msg="+ local -r NAME=cilium_vxlan" subsys=datapath-loader
level=warning msg="+ ip link set cilium_vxlan up" subsys=datapath-loader
level=warning msg="+ '[' '<nil>' '!=' '<nil>' ']'" subsys=datapath-loader
level=warning msg="+ '[' 10.0.9.61 '!=' '<nil>' ']'" subsys=datapath-loader
level=warning msg="+ echo 1" subsys=datapath-loader
level=warning msg="+ echo 0" subsys=datapath-loader
level=warning msg="+ echo 1" subsys=datapath-loader
level=warning msg="+ echo 0" subsys=datapath-loader
level=warning msg="++ cat /sys/class/net/cilium_vxlan/ifindex" subsys=datapath-loader
level=warning msg="+ ENCAP_IDX=14" subsys=datapath-loader
level=warning msg="+ sed -i '/^#.*ENCAP_IFINDEX.*$/d' /var/run/cilium/state/globals/node_config.h" subsys=datapath-loader
level=warning msg="+ echo '#define ENCAP_IFINDEX 14'" subsys=datapath-loader
level=warning msg="+ CALLS_MAP=cilium_calls_overlay_2" subsys=datapath-loader
level=warning msg="+ COPTS='-DSECLABEL=2 -DFROM_ENCAP_DEV=1'" subsys=datapath-loader
level=warning msg="+ '[' true = true ']'" subsys=datapath-loader
level=warning msg="+ COPTS='-DSECLABEL=2 -DFROM_ENCAP_DEV=1 -DDISABLE_LOOPBACK_LB'" subsys=datapath-loader
level=warning msg="+ bpf_load cilium_vxlan '-DSECLABEL=2 -DFROM_ENCAP_DEV=1 -DDISABLE_LOOPBACK_LB' ingress bpf_overlay.c bpf_overlay.o from-overlay cilium_calls_overlay_2" subsys=datapath-loader
level=warning msg="+ DEV=cilium_vxlan" subsys=datapath-loader
level=warning msg="+ OPTS='-DSECLABEL=2 -DFROM_ENCAP_DEV=1 -DDISABLE_LOOPBACK_LB'" subsys=datapath-loader
level=warning msg="+ WHERE=ingress" subsys=datapath-loader
level=warning msg="+ IN=bpf_overlay.c" subsys=datapath-loader
level=warning msg="+ OUT=bpf_overlay.o" subsys=datapath-loader
level=warning msg="+ SEC=from-overlay" subsys=datapath-loader
level=warning msg="+ CALLS_MAP=cilium_calls_overlay_2" subsys=datapath-loader
level=warning msg="++ ip link show cilium_vxlan" subsys=datapath-loader
level=warning msg="++ grep ether" subsys=datapath-loader
level=warning msg="++ awk '{print $2}'" subsys=datapath-loader
level=warning msg="+ NODE_MAC=e6:1e:5f:e4:16:3c" subsys=datapath-loader
level=warning msg="++ mac2array e6:1e:5f:e4:16:3c" subsys=datapath-loader
level=warning msg="++ echo '{0xe6,0x1e,0x5f,0xe4,0x16,0x3c}'" subsys=datapath-loader
level=warning msg="+ NODE_MAC='{.addr={0xe6,0x1e,0x5f,0xe4,0x16,0x3c}}'" subsys=datapath-loader
level=warning msg="+ OPTS='-DSECLABEL=2 -DFROM_ENCAP_DEV=1 -DDISABLE_LOOPBACK_LB -DNODE_MAC={.addr={0xe6,0x1e,0x5f,0xe4,0x16,0x3c}} -DCALLS_MAP=cilium_calls_overlay_2'" subsys=datapath-loader
level=warning msg="+ bpf_compile bpf_overlay.c bpf_overlay.o obj '-DSECLABEL=2 -DFROM_ENCAP_DEV=1 -DDISABLE_LOOPBACK_LB -DNODE_MAC={.addr={0xe6,0x1e,0x5f,0xe4,0x16,0x3c}} -DCALLS_MAP=cilium_calls_overlay_2'" subsys=datapath-loader
level=warning msg="+ IN=bpf_overlay.c" subsys=datapath-loader
level=warning msg="+ OUT=bpf_overlay.o" subsys=datapath-loader
level=warning msg="+ TYPE=obj" subsys=datapath-loader
level=warning msg="+ EXTRA_OPTS='-DSECLABEL=2 -DFROM_ENCAP_DEV=1 -DDISABLE_LOOPBACK_LB -DNODE_MAC={.addr={0xe6,0x1e,0x5f,0xe4,0x16,0x3c}} -DCALLS_MAP=cilium_calls_overlay_2'" subsys=datapath-loader
level=warning msg="+ clang -O2 -target bpf -std=gnu89 -nostdinc -emit-llvm -g -Wall -Wextra -Werror -Wshadow -Wno-address-of-packed-member -Wno-unknown-warning-option -Wno-gnu-variable-sized-type-not-at-end -Wdeclaration-after-statement -Wimplicit-int-conversion -Wenum-conversion -I. -I/run/cilium/state/globals -I/var/lib/cilium/bpf -I/var/lib/cilium/bpf/include -D__NR_CPUS__=16 -DENABLE_ARP_RESPONDER=1 -DSECLABEL=2 -DFROM_ENCAP_DEV=1 -DDISABLE_LOOPBACK_LB '-DNODE_MAC={.addr={0xe6,0x1e,0x5f,0xe4,0x16,0x3c}}' -DCALLS_MAP=cilium_calls_overlay_2 -c /var/lib/cilium/bpf/bpf_overlay.c -o -" subsys=datapath-loader
level=warning msg="+ llc -march=bpf -mcpu=v2 -mattr=dwarfris -filetype=obj -o bpf_overlay.o" subsys=datapath-loader
level=warning msg="+ tc qdisc replace dev cilium_vxlan clsact" subsys=datapath-loader
level=warning msg="++ tc filter show dev cilium_vxlan ingress" subsys=datapath-loader
level=warning msg="++ grep -v 'pref 1 bpf chain 0 $\\|pref 1 bpf chain 0 handle 0x1'" subsys=datapath-loader
level=warning msg="+ '[' -z '' ']'" subsys=datapath-loader
level=warning msg="+ cilium bpf migrate-maps -s bpf_overlay.o" subsys=datapath-loader
level=warning msg="+ tc filter replace dev cilium_vxlan ingress prio 1 handle 1 bpf da obj bpf_overlay.o sec from-overlay" subsys=datapath-loader
level=warning msg="libbpf: load bpf program failed: ERROR: strerror_r(524)=22" subsys=datapath-loader
level=warning msg="libbpf: -- BEGIN DUMP LOG ---" subsys=datapath-loader
level=warning msg="libbpf: Skipped 19998129 bytes from the verifier log" subsys=datapath-loader
level=warning msg="libbpf: " subsys=datapath-loader
level=warning msg=" = 1" subsys=datapath-loader
level=warning msg="318: (b7) r3 = 0" subsys=datapath-loader
level=warning msg="; tuple.dport = icmphdr.un.echo.id;" subsys=datapath-loader
level=warning msg="319: (69) r1 = *(u16 *)(r10 -92)" subsys=datapath-loader
level=warning msg="320: (05) goto pc-123" subsys=datapath-loader
level=warning msg="; " subsys=datapath-loader
level=warning msg="198: (6b) *(u16 *)(r10 -144) = r1" subsys=datapath-loader
level=warning msg="199: (6b) *(u16 *)(r10 -142) = r3" subsys=datapath-loader
level=warning msg="last_idx 199 first_idx 198" subsys=datapath-loader
level=warning msg="regs=8 stack=0 before 198: (6b) *(u16 *)(r10 -144) = r1" subsys=datapath-loader
level=warning msg=" R0_w=inv4294967162 R1_rw=inv(id=0,umax_value=65535,var_off=(0x0; 0xffff)) R2_w=inv1 R3_rw=invP0 R5_w=inv(id=0) R6=ctx(id=0,off=0,imm=0) R8=pkt(id=0,off=0,r=34,imm=0) R9=invP0 R10=fp0 fp-96=mmmmmmmm fp-144=??mm0000 fp-152=mmmmmmmm fp-160=????0000 fp-168=00000000 fp-176=00000000 fp-184=00000000 fp-192=00000000 fp-200=00000000 fp-208=00000000 fp-216=0000mmmm fp-224=0000000m fp-232=ctx fp-240=mmmmmmmm fp-248=mmmmmmmm fp-256=00000000" subsys=datapath-loader
level=warning msg="parent didn't have regs=8 stack=0 marks" subsys=datapath-loader
level=warning msg="last_idx 320 first_idx 211" subsys=datapath-loader
level=warning msg="regs=8 stack=0 before 320: (05) goto pc-123" subsys=datapath-loader
level=warning msg="regs=8 stack=0 before 319: (69) r1 = *(u16 *)(r10 -92)" subsys=datapath-loader
level=warning msg="regs=8 stack=0 before 318: (b7) r3 = 0" subsys=datapath-loader
level=warning msg="; if (dir == NAT_DIR_EGRESS &&" subsys=datapath-loader
level=warning msg="200: (15) if r5 == 0x1 goto pc+27" subsys=datapath-loader
level=warning msg=" R0=inv4294967162 R1=inv(id=0,umax_value=65535,var_off=(0x0; 0xffff)) R2=inv1 R3=invP0 R5=inv(id=0) R6=ctx(id=0,off=0,imm=0) R8=pkt(id=0,off=0,r=34,imm=0) R9=invP0 R10=fp0 fp-96=mmmmmmmm fp-144=??mm00mm fp-152=mmmmmmmm fp-160=????0000 fp-168=00000000 fp-176=00000000 fp-184=00000000 fp-192=00000000 fp-200=00000000 fp-208=00000000 fp-216=0000mmmm fp-224=0000000m fp-232=ctx fp-240=mmmmmmmm fp-248=mmmmmmmm fp-256=00000000" subsys=datapath-loader
level=warning msg="201: (55) if r5 != 0x0 goto pc+30" subsys=datapath-loader
level=warning msg=" R0=inv4294967162 R1=inv(id=0,umax_value=65535,var_off=(0x0; 0xffff)) R2=inv1 R3=invP0 R5=inv0 R6=ctx(id=0,off=0,imm=0) R8=pkt(id=0,off=0,r=34,imm=0) R9=invP0 R10=fp0 fp-96=mmmmmmmm fp-144=??mm00mm fp-152=mmmmmmmm fp-160=????0000 fp-168=00000000 fp-176=00000000 fp-184=00000000 fp-192=00000000 fp-200=00000000 fp-208=00000000 fp-216=0000mmmm fp-224=0000000m fp-232=ctx fp-240=mmmmmmmm fp-248=mmmmmmmm fp-256=00000000" subsys=datapath-loader
level=warning msg="; " subsys=datapath-loader
level=warning msg="202: (18) r0 = 0xffffff53" subsys=datapath-loader
level=warning msg="last_idx 204 first_idx 198" subsys=datapath-loader
level=warning msg="regs=1 stack=0 before 202: (18) r0 = 0xffffff53" subsys=datapath-loader
level=warning msg="last_idx 204 first_idx 198" subsys=datapath-loader
level=warning msg="regs=4 stack=0 before 202: (18) r0 = 0xffffff53" subsys=datapath-loader
level=warning msg="regs=4 stack=0 before 201: (55) if r5 != 0x0 goto pc+30" subsys=datapath-loader
level=warning msg="regs=4 stack=0 before 200: (15) if r5 == 0x1 goto pc+27" subsys=datapath-loader
level=warning msg="regs=4 stack=0 before 199: (6b) *(u16 *)(r10 -142) = r3" subsys=datapath-loader
level=warning msg="regs=4 stack=0 before 198: (6b) *(u16 *)(r10 -144) = r1" subsys=datapath-loader
level=warning msg=" R0_w=inv4294967162 R1_rw=inv(id=0,umax_value=65535,var_off=(0x0; 0xffff)) R2_rw=invP1 R3_rw=invP0 R5_rw=inv(id=0) R6_r=ctx(id=0,off=0,imm=0) R8=pkt(id=0,off=0,r=34,imm=0) R9=invP0 R10=fp0 fp-96=mmmmmmmm fp-144=??mm0000 fp-152=mmmmmmmm fp-160_r=????0000 fp-168_r=00000000 fp-176_r=00000000 fp-184_r=00000000 fp-192_r=00000000 fp-200_r=00000000 fp-208_r=00000000 fp-216_r=0000mmmm fp-224_r=0000000m fp-232_r=ctx fp-240_r=mmmmmmmm fp-248=mmmmmmmm fp-256=00000000" subsys=datapath-loader
level=warning msg="parent didn't have regs=4 stack=0 marks" subsys=datapath-loader
level=warning msg="last_idx 320 first_idx 211" subsys=datapath-loader
level=warning msg="regs=4 stack=0 before 320: (05) goto pc-123" subsys=datapath-loader
level=warning msg="regs=4 stack=0 before 319: (69) r1 = *(u16 *)(r10 -92)" subsys=datapath-loader
level=warning msg="regs=4 stack=0 before 318: (b7) r3 = 0" subsys=datapath-loader
level=warning msg="regs=4 stack=0 before 317: (b7) r2 = 1" subsys=datapath-loader
level=warning msg="204: safe" subsys=datapath-loader
level=warning subsys=datapath-loader
level=warning msg="from 201 to 232: R0=inv4294967162 R1=inv(id=0,umax_value=65535,var_off=(0x0; 0xffff)) R2=inv1 R3=invP0 R5=inv(id=0) R6=ctx(id=0,off=0,imm=0) R8=pkt(id=0,off=0,r=34,imm=0) R9=invP0 R10=fp0 fp-96=mmmmmmmm fp-144=??mm00mm fp-152=mmmmmmmm fp-160=????0000 fp-168=00000000 fp-176=00000000 fp-184=00000000 fp-192=00000000 fp-200=00000000 fp-208=00000000 fp-216=0000mmmm fp-224=0000000m fp-232=ctx fp-240=mmmmmmmm fp-248=mmmmmmmm fp-256=00000000" subsys=datapath-loader
level=warning msg="; if (dir == NAT_DIR_INGRESS && (dport < target->min_port || dport > target->max_port))" subsys=datapath-loader
level=warning msg="232: (bf) r2 = r10" subsys=datapath-loader
level=warning msg="; " subsys=datapath-loader
level=warning msg="233: (07) r2 += -152" subsys=datapath-loader
level=warning msg="; return map_lookup_elem(map, tuple);" subsys=datapath-loader
level=warning msg="234: (18) r1 = 0xffff93fc32923000" subsys=datapath-loader
level=warning msg="236: (85) call bpf_map_lookup_elem#1" subsys=datapath-loader
level=warning msg="237: safe" subsys=datapath-loader
level=warning subsys=datapath-loader
level=warning msg="from 200 to 228: safe" subsys=datapath-loader
level=warning msg="last_idx 8062 first_idx 211" subsys=datapath-loader
level=warning msg="regs=1 stack=0 before 217: (c5) if r1 s< 0x0 goto pc+7844" subsys=datapath-loader
level=warning msg="regs=1 stack=0 before 215: (18) r0 = 0xffffff7a" subsys=datapath-loader
level=warning subsys=datapath-loader
level=warning msg="from 217 to 8062: safe" subsys=datapath-loader
level=warning msg="last_idx 8062 first_idx 0" subsys=datapath-loader
level=warning msg="regs=1 stack=0 before 163: (2d) if r2 > r1 goto pc+7898" subsys=datapath-loader
level=warning msg="regs=1 stack=0 before 161: (18) r0 = 0xffffff7a" subsys=datapath-loader
level=warning subsys=datapath-loader
level=warning msg="from 163 to 8062: safe" subsys=datapath-loader
level=warning msg="processed 63542 insns (limit 1000000) max_states_per_insn 14 total_states 4146 peak_states 1929 mark_read 411" subsys=datapath-loader
level=warning subsys=datapath-loader
level=warning msg="libbpf: -- END LOG --" subsys=datapath-loader
level=warning msg="libbpf: failed to load program 'tail_nodeport_nat_ipv4'" subsys=datapath-loader
level=warning msg="libbpf: failed to load object 'bpf_overlay.o'" subsys=datapath-loader
level=warning msg="Unable to load program" subsys=datapath-loader
level=warning msg="+ cilium bpf migrate-maps -e bpf_overlay.o -r 1" subsys=datapath-loader
level=warning msg="+ return 1" subsys=datapath-loader
level=fatal msg="Error while creating daemon" error="error while initializing daemon: failed while reinitializing datapath: exit status 1" subsys=daemon
```

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by binhnguyenduc** [comment]
I have done some more digging and it seems to be some issue with setting up the cilium vxlan tunnel

**Comment by reyvonger** [comment]
same error on talos.dev

**Comment by binhnguyenduc** [comment]
I have pinpointed the problem to be of the Linux kernel. 

The kernel giving above error: Oracle Linux Server 7.9, with Unbreakable Enterprise Kernel 5.4.17-2136.311.6.1.el7uek.x86_64

The kernel where Cilium started without issue: Oracle Linux Server 7.9, with Unbreakable Enterprise Kernel 5.4.17-2136.304.4.2.el7uek.x86_64

**Comment by brb** [comment]
@binhnguyenduc Thanks. Do you have a changelog diff between `5.4.17-2136.311.6.1.el7uek.x86_64` and `5.4.17-2136.304.4.2.el7uek.x86_64`?
